staging/src/k8s.io/csi-translation-lib/plugins/aws_ebs.go

```diff
@@ -78,8 +78,9 @@ func (t *awsElasticBlockStoreCSITranslator) TranslateInTreeInlineVolumeToCSI(vol
 	}
 	pv := &v1.PersistentVolume{
 		ObjectMeta: metav1.ObjectMeta{
-			// A.K.A InnerVolumeSpecName required to match for Unmount
-			Name: volume.Name,
+			// Must be unique per disk as it is used as the unique part of the
+			// staging path
+			Name: fmt.Sprintf("%s-%s", AWSEBSDriverName, ebsSource.VolumeID), 
```

Comment:
Does the driver name have any characters like "/" that would fail validation for this object?

Reply:
or exceed length?

Reply:
they all have a lot of `.`'s that could be a problem. Checkin gnow

Reply:
confirmed none of them have any validation failing characters

Reply:
I clarified with @davidz627 that these objects are not actually persisted in the apiserver. The only place the name is persisted is in the staging path on the host, and we shouldn't have name length problems there.